packages/core/src/components/button/button.scss

```diff
@@ -37,9 +37,13 @@
       background-color: helpers.color('background-action');
       color: helpers.color('content-on-action');
 
-      &:hover {
-        background-color: helpers.color('background-action-hover');
+      /* stylelint-disable max-nesting-depth */
```

Comment:
[minor] didn't `stylelint-disable-next-line` work? It would cut these comments in half

Reply:
If I use `stylelint-disable-next-line` I need to write :

```
      /* stylelint-disable-next-line max-nesting-depth */
      @media (hover: hover) {
        /* stylelint-disable-next-line max-nesting-depth */
        &:hover {
          background-color: helpers.color('background-negative-subtle-hover');
        }
      }
```

So I found that using disable/enable for more clear

Reply:
Ah because we ignore pseudo-classes, so now it's 3 depth...
Alright, looks better the way it is then, thanks for clarifying